client/pages/BookClubs/index.jsx

```diff
@@ -130,28 +130,50 @@ export default function BookClubs() {
     return (
       <div style={{ position: 'relative' }}>
         <div className="club-btns">
-          <div className="prev">
-            <button onClick={handlePrev} type="button">
-              <ArrowBackIosIcon />
-            </button>
-          </div>
-          <div className="next">
-            <button onClick={handleNext} type="button">
-              <ArrowForwardIosIcon />
-            </button>
-          </div>
+          {myBookClubs.length === 0 ? (
```

Comment:
Is it ever possible for `myBookClubs` to be `null` or `undefined`?

Reply:
If so, I recommend replacing `myBookClubs.length` with `myBookClubs?.length`

Reply:
myBookClubs state gets set with `data.results` returned from myBookclubs endpoint which always returns an empty array when there's no value.